doc/source/user_guide/visualization.rst

```diff
@@ -1641,3 +1641,45 @@ when plotting a large number of points.
    :suppress:
 
     plt.close('all')
+
+Plotting backends
+-----------------
+
+Starting in 0.25 pandas can be extended with third-party plotting backends. The
+main idea is letting users select a plotting backend different than the provided
+one based on Matplotlib. For example:
+
+.. code-block:: python
+
+    >>> pd.set_option('plotting.backend', 'backend.module')
+    >>> pd.Series([1, 2, 3]).plot()
+
+This would be more or less equivalent to:
+
+.. code-block:: python
+
+    >>> import backend.module
+    >>> backend.module.plot(pd.Series([1, 2, 3]))
+
+The backend module can then use other visualization tools (Bokeh, Altair,...)
+to generate the plots.
```

Comment:
It might be interesting to add here some actual example of libraries that do this. I am aware of pandas-bokeh (https://github.com/PatrikHlobil/Pandas-Bokeh) and hvplot (https://hvplot.holoviz.org/user_guide/Pandas_API.html), but there are probably others. (cc @datapythonista)

(it's always difficult to be complete when starting to list things though ..)

Reply:
I think it's just Altair that adapted it (not sure if there is a released version). I discussed with plotly, and they were interested on implementing this, but I'm not aware of any work from them.

Reply:
Made all the changes